### PR TITLE
Tidying scenarios documentation.

### PR DESCRIPTION
**Pull request by abdullahgarcia**
The scenarios documentation lacked a format.

A new format has been introduced to ensure full understanding of the scenario and the tasks to be performed by the user.

### CODE REVIEW

**Comment by jondkent** [comment]
looks fine to merge

**Comment by jondkent** [comment]
@s-irvine  , can you review and approve if happy,  thanks

File: simulation-scripts/scenario/container-ambush/challenge.txt [hunk 1]
```diff
@@ -1,5 +1,8 @@
-Task1: During a security audit you have noticed an unpatched vulnerability in a CI build agent in your build farm.  This
-could potentially lead to the expose of sensitive data in the attached databases.  Demonstrate whether this is possible
-by retrieving x data from the database.
+Scenario: During a security audit you have noticed an unpatched vulnerability in a CI build agent in your build farm.  This
```
Comment (s-irvine) [inline simulation-scripts/scenario/container-ambush/challenge.txt]: These two lines should probably be one

File: simulation-scripts/scenario/network-feint/challenge.txt [hunk 2]
```diff
@@ -1,6 +1,13 @@
-Your users are reporting odd results when they query your application nodeport of 32123. They are expecting to see the nginx welcome page, but 1/3 of the time they get an empty response. As a security engineer you are concerned because the application itself appears to be working normally and the only other pod in this namespace is a database with minimal security configuration due to business requirements.
-Can you interact with the nodeport to find a secret key, therefore proving there is an error in configuration for this namespace?
+Scenario: Your users are reporting odd results when they query your application NodePort: 32123.
```
Comment (s-irvine) [inline simulation-scripts/scenario/network-feint/challenge.txt]:
```suggestion
Scenario: Your users are reporting odd results when they query your application NodePort: "$MASTER_IP_ADDRESS":32123.
```

File: simulation-scripts/scenario/rbac-flanking-maneuver/challenge.txt [hunk 3]
```diff
@@ -1,6 +1,7 @@
-Task 1: You have been asked to perform a security test on an application hosted in Kubernetes. To make the test as close
-to reality as possible you have only been told that the application is exposed as a NodePort service on IPaddress
-x.x.x.x
+Scenario: You have been asked to perform a security test on an application hosted in Kubernetes.
+To make the test as close to reality as possible you have only been told that the application is exposed as a NodePort service on IP address x.x.x.x.
```
Comment (s-irvine) [inline simulation-scripts/scenario/rbac-flanking-maneuver/challenge.txt]:
```suggestion
To make the test as close to reality as possible you have only been told that the application is exposed as a NodePort service on IP address $MASTER_IP_ADDRESS
```

File: simulation-scripts/scenario/container-ambush/challenge.txt (hunk 1 above)
Comment (abdullahgarcia) [reply simulation-scripts/scenario/container-ambush/challenge.txt]: Addressed in last commit.

File: simulation-scripts/scenario/rbac-flanking-maneuver/challenge.txt (hunk 3 above)
Comment (abdullahgarcia) [reply simulation-scripts/scenario/rbac-flanking-maneuver/challenge.txt]: Addressed in last commit.

File: simulation-scripts/scenario/network-feint/challenge.txt (hunk 2 above)
Comment (abdullahgarcia) [reply simulation-scripts/scenario/network-feint/challenge.txt]: Addressed in last commit.

**Pull request merged**